### PR TITLE
nload: add page

### PR DESCRIPTION
**Pull request by jakevossen5**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/nload.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# nload
+
+> A tool for visualizing network usage in the terminal.
+> More information: <https://github.com/rolandriegel/nload>.
+
+- View all network traffic (arrow keys to switch interfaces):
```
Comment (bl-ue) [inline pages/common/nload.md:6]:
```suggestion
- View all network traffic (use the arrow keys to switch interfaces):
```

File: pages/common/nload.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`nload`
+
+- View network traffic on a particular interfaces (arrow keys to switch interfaces):
```
Comment (bl-ue) [inline pages/common/nload.md:10]:
```suggestion
- View network traffic on a particular interfaces (use the arrow keys to switch interfaces):
```

**Review by bl-ue — APPROVED** [review]

File: pages/common/nload.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# nload
+
+> A tool for visualizing network usage in the terminal.
+> More information: <https://github.com/rolandriegel/nload>.
+
+- View all network traffic (use the arrow keys to switch interfaces):
+
+`nload`
+
+- View network traffic on a particular interfaces (use the arrow keys to switch interfaces):
```
Comment (marchersimon) [inline pages/common/nload.md:10]:
```suggestion
- View the network traffic on specific interfaces (use the arrow keys to switch interfaces):
```

Comment (marchersimon) [reply pages/common/nload.md:10]: Sure. Is this a general rule or just in this example?

Comment (bl-ue) [reply pages/common/nload.md:10]: Just in this example 👍🏻

Comment (bl-ue) [reply pages/common/nload.md:10]: If you feel strongly otherwise, we don't need at all.

**Pull request merged**